code/gb/usamerm/res/pin/Makefile

```diff
@@ -8,6 +8,8 @@ all: $(EXE)
 conv: conv.o gfx.o font.o
 	gcc -o $@ $^ $(shell sdl-config --libs) -lm
 
+pinmap: pinmap.c
```

Comment:
pinmap doesn't need -lm
sdl-config includes + libs don't hurt

Reply:
It crashes on Windows if you add SDL. Removed -lm in next commit.